test/evergreen.yml

```diff
@@ -2386,6 +2386,24 @@ tasks:
           times: 1       # No of times to run the loop
           no_of_procs: 10 # No of processes to run in the background
 
+  - name: performance-test
+    tags: ["performance-test-1"]
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          configure_env_vars:
+            CXX=/opt/mongodbtoolchain/v3/bin/g++
+            PATH=/opt/mongodbtoolchain/v3/bin:$PATH ADD_CFLAGS="-ggdb -fPIC"
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/bench/workgen/runner"
+          script: |
+            set -o errexit
+            set -o verbose
+            export "PATH=/opt/mongodbtoolchain/v3/bin:$PATH"
+            ./workgen_perf_check.sh many-dhandle-stress.py output.log 50 500 500
+
```

Comment:
Can we move this new block of code down, outside of the stress task definitions?

Reply:
I just realized it's a new perf task with stress as part of the script name. Anyway, I feel it's more of a 'perf' test and should be set in its own 'section' in the evergreen configuration file.